src/orm_test.cr

```diff
@@ -1,7 +1,9 @@
 require "benchmark"
 require "./orm_test/*"
 
-DATABASE = {host: "localhost", name: "crystal_orm_test", user: "postgres"}
+USERNAME = `whoami`.chomp
```

Comment:
I think for this I'd rather keep `postgres` as default, then maybe add ENV override. I think `postgres` will be a more likely user especially for people using docker (like me :grimacing: )... So maybe something like

```suggestion
USERNAME = ENV.fetch("DB_USER", "postgres")
```
```

Reply:
I changed this in #8 so I could run the specs.